src/nanosvg/arc_to_cubic.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Convert SVG Path's elliptical arcs to Bezier curves.
```

Comment:
might be worth commenting on why this is desirable

Reply:
well, in general because bezier are easier to consume than arcs, and cubic beziers can approximate arcs just fine.
And if we don't convert them, they will be converted anyway as we process paths with Skia, but to a lesser form (i.e. first internally stored as conics, and then approximated to quadratic beziers which are less flexible, more verbose than cubics).
I would even go as far as banning arcs altogether for our `nsvg` format. Also `usvg` automatically converts arcs to cubic https://docs.rs/usvg/0.9.0/usvg/struct.PathData.html#method.push_arc_to
usvg only allows moveTo, lineTo, cubicTo and closePath commands